### DISCUSSION TITLE
cannot allocate vector of size 314.6 Gb

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#283] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78478403/187057295-8f4826a6-5b16-41f0-b95f-ae7ea204432b.png)
ini kenapa ya kak.?

**Comment by onlyphantom** [comment, marked as answer]
How many RAM do you have on your machine?

If you're on PC / Windows, [try the following](https://stackoverflow.com/questions/5171593/r-memory-management-cannot-allocate-vector-of-size-n-mb):

- Before opening R, open the Windows Resource Monitor (Ctrl-Alt-Delete / Start Task Manager / Performance tab / click on bottom button 'Resource Monitor' / Memory tab)
- you will see how much RAM memory us already used before you open R, and by which applications. For example you might see that 1.6 GB of the total 4GB are used. In that example you will only be able to get 2.4 GB for R

This is a massive oversimplification but the message is essentially saying that the computer do not has the required RAM space to hold a vector that is going to "eat up" 314 Gb

**Reply by daryraa** [reply]
ram yang saya punya 8 gb mas

**Reply by onlyphantom** [reply]
Use gc() to clear everything (garbage collection) and create only that specific model. :)

**Reply by daryraa** [reply]
okei terima kasih mas
